packages/@apphosting/common/src/index.ts

```diff
@@ -1,5 +1,58 @@
 import { spawn } from "child_process";
 
+// Output bundle specifications to be written to bundle.yaml
+export interface OutputBundle {
```

Comment:
So one quick thought: I thought OutputBundle was the name for the entirety of the framework build process which includes:
* bundle.yaml
* static assets
* source build artifacts

Is it accurate to name this outputbundle when we're actually defining the metadata for the output bundle?